### DISCUSSION TITLE
『浅入浅出』MongoDB 和 WiredTiger · /mongodb-wiredtiger

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#250] [Announcements]
https://draveness.me/mongodb-wiredtiger

**Comment by zhuoyikang** [comment]
这图画的好好看用什么画的呢？

**Comment by fenghaojiang** [comment]
“如何表示数据模型之间的关系其实是需要开发者需要仔细考虑的”有个typo，两个“需要”了

---
*2020-11-28 UPDATES 已修复*

**Comment by zhenliuxc** [comment]
> 前四位代表当前 `_id` 生成时的 Unix 时间戳，在这之后是三位的机器标识符和两位的处理器标识符，最后是一个三位的计数器，初始值就是一个随机数；

“前四位”，第一次看有点疑惑，“前四个字节”能更好减少歧义。

**Comment by MaeserYang** [comment]
wiredTiger底层是否有用到mmap实现呢？
